### PR TITLE
added packages/rankers/rankers.1.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**
Do not merge: I will rename to 1.0.0 if everything works.

### CODE REVIEW

File: packages/rankers/rankers.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "base-unix"
+  "batteries"
+  "bst"
+  "conf-gnuplot"
+  "cpm"
+  "dolog" {>= "4.0.0"}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/rankers/rankers.1.0.0/opam]:
```suggestion
  "dune" {>= "1.6"}
```
According to your `dune-project` file, this package requires at least dune 1.6

**Comment by kit-ty-kate** [comment]
Good to merge once fixed

**Comment by UnixJunkie** [comment]
Fixed, thanks.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**